CONTRIBUTING.md

```diff
@@ -34,7 +34,43 @@ There are 3 ways to test your changes:
 - Manual testing
   - Run `./maestro` instead of `maestro` to use your local code.
 - Unit tests
-  - All the other tests in the projects. Run them via `./gradlew test` (or from IDE) 
+  - All the other tests in the projects. Run them via `./gradlew test` (or from IDE)
+
+If you made changes to the CLI, rebuilt it with `./gradlew :maestro-cli:installDist`. This will generate a startup shell
+script in `./maestro-cli/build/install/maestro/bin/maestro`. Use it instead of globally installed `maestro`.
```

Comment:
nit: You can recommend running `./maestro test` as well

Reply:
That script always triggers a Gradle build, which I personally found annoying because it was slowing me down on repeated CLI runs.